app/models/concerns/comment/publishing.rb

```diff
@@ -26,9 +26,10 @@ def to_event_stream
         project_id: project_id,
         section: section,
         body: body,
-        user_id: Digest::SHA1.hexdigest(user_id.to_s),
+        user_id: user_id.to_s,
```

Comment:
i think so, we're publishing this data already via panoptes and it's not private data (accessible via the api)